hazelcast-client/src/main/java/com/hazelcast/client/impl/HazelcastClientProxy.java

```diff
@@ -293,7 +293,7 @@ public SerializationService getSerializationService() {
     protected HazelcastClientInstanceImpl getClient() {
         final HazelcastClientInstanceImpl c = client;
         if (c == null || !c.getLifecycleService().isRunning()) {
-            throw new HazelcastInstanceNotActiveException();
+            throw new HazelcastClientNotActiveException("Client is not active.");
```

Comment:
Seems we should also change this one to `HazelcastClientNotActiveException`: https://github.com/hazelcast/hazelcast/blob/master/hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/AbstractClientInternalCacheProxy.java#L694

Reply:
Changed.